modules/budget/variables.tf

```diff
@@ -53,6 +53,44 @@ variable "services" {
   default     = null
 }
 
+variable "calendar_period" {
```

Comment:
Lets plumb some of these inputs to the main module
https://github.com/terraform-google-modules/terraform-google-project-factory/blob/9410ca40050668535068968a1a18a3ecfda915d7/main.tf#L90